django/db/models/expressions.py

```diff
@@ -2140,3 +2140,72 @@ class ValueRange(WindowFrame):
 
     def window_frame_start_end(self, connection, start, end):
         return connection.ops.window_frame_range_start_end(start, end)
+
+
+class RowTuple(ExpressionList):
```

Comment:
This one will be useful for generic composite field support as well #18056, #17279 fyi @csirmazbendeguz